include/SDL3/SDL_gpu.h

```diff
@@ -4177,6 +4177,27 @@ extern SDL_DECLSPEC Uint32 SDLCALL SDL_CalculateGPUTextureFormatSize(
     Uint32 height,
     Uint32 depth_or_layer_count);
 
+
+#define SDL_PROP_GPU_DEVICE_PHYSICAL_NAME_STRING        "SDL.gpu.device.physical.name"
+#define SDL_PROP_GPU_DEVICE_DRIVER_VERSION_STRING       "SDL.gpu.device.driver.version"
+
+/**
+ * Get readonly metadata properties of an existing GPU device.
+ * 
+ * Currently available properties:
+ * 
+ * - `SDL_PROP_GPU_DEVICE_PHYSICAL_NAME_STRING`: The physical name of the GPU device, as reported by the system driver.
```

Comment:
Is the `PHYSICAL` part redundant? What other device name could it be referring to? Maybe `ADAPTER` instead?